Dockerfile

```diff
@@ -1,2 +1,14 @@
-FROM golang:onbuild
+FROM golang:alpine
```

Comment:
As discussed, better to do this for a smaller / cleaner image : https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds

Reply:
✓